app/views/idv/verify_info/show.html.erb

```diff
@@ -90,25 +90,15 @@ locals:
     </div>
   </div>
   <div class="margin-top-5">
-    <%= render SpinnerButtonComponent.new(
+    <%= render ButtonComponent.new(
```

Comment:
I believe we need to make this back into a `SpinnerButtonComponent`, but presumably we can save that for the async story?

Reply:
Yes, that's the idea. We are working on that part of things in the `sonia-lg-7401` branch at the moment